packages/app/src/components/__tests__/DBSearchPage.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { getDefaultSourceId } from '../../DBSearchPage';
+
+describe('getDefaultSourceId', () => {
+  it('returns empty string if sources is undefined', () => {
+    expect(getDefaultSourceId(undefined, undefined)).toBe('');
+  });
+
+  it('returns empty string if sources is empty', () => {
+    expect(getDefaultSourceId([], undefined)).toBe('');
+  });
```

Comment:
Does it make sense to check a condition where the source array is empty but the last source id is a string value?